config/lib/log.js

```diff
@@ -11,6 +11,7 @@ module.exports = winston.createLogger({
   format: winston.format.combine(
     winston.format.colorize(),
     winston.format.timestamp(),
+    winston.format.splat(),
```

Comment:
Allows the usage of `%s` in loggers to pass in varargs at the end to be parameterized.

This probably wouldn't really be a thing if we were using ES6 and had backtick interpolation.